pkg/internal/request/span_getters.go

```diff
@@ -34,8 +34,12 @@ func SpanOTELGetters(name attr.Name) (attributes.Getter[*Span, attribute.KeyValu
 		getter = func(s *Span) attribute.KeyValue { return semconv.HTTPRoute(s.Route) }
 	case attr.HTTPUrlPath:
 		getter = func(s *Span) attribute.KeyValue { return HTTPUrlPath(s.Path) }
+	case attr.SrcName:
+		fallthrough
```

Comment:
I'd say the resolution is a bit more complex. Here the Src/Dst name would resolve always to the IP address, but we should prioritize first the K8s pod name or the host name, as for Network Flows.